src/worldofzuul/CommandWords.java

```diff
@@ -7,31 +7,51 @@
  * @version 2006.03.30
  */
 
+
 public class CommandWords {
     private HashMap<String, CommandWord> validCommands;
-
+    
+    /**
+     * Creating an instance of CommandWords
+     * Creating an empty list 
+     * Validating if the commandword is known
+     * Storing if its valid
+     */
     public CommandWords() {
-        validCommands = new HashMap<String, CommandWord>();
-        for(CommandWord command : CommandWord.values()) {
-            if(command != CommandWord.UNKNOWN) {
-                validCommands.put(command.toString(), command);
+        validCommands = new HashMap<String, CommandWord>(); 
+        for(CommandWord command : CommandWord.values()) {   
+            if(command != CommandWord.UNKNOWN) {            
+                validCommands.put(command.toString(), command); 
```

Comment:
Tabs eller spaces brude blive fjernet. 😄

Reply:
Fixed in 6702a6201bb065bac78d30ca9bd4c3c467c64a6e